.github/workflows/CI.yml

```diff
@@ -27,8 +27,8 @@ jobs:
           sudo apt install -y libmpfr-dev
           sudo apt install -y autoconf
           sudo apt install -y libtool-bin
-          sudo apt install -y gcovr
           gcc --version
+          gcov --version
```

Comment:
So is gcovr used after all? But you also remove the code installing it. I am confused :-)

Reply:
Oh, I thought `gcov` and `gcovr` was two different things... What am I supposed to use? I have never done this before :-)

Reply:
No, all is right, I am just stupid and misread `gcov` :-(. Sorry for the confusion!!!